python/lib/sift_py/data_import/ch10.py

```diff
@@ -71,7 +71,7 @@ def __next__(self) -> str:
 class Ch10UploadService(CsvUploadService):
     """Service to upload ch10 files."""
 
-    def upload_ch10(
+    def upload(  # type: ignore
```

Comment:
What was the error we're suppressing here?

Reply:
It was complaining about overriding the parent function with a different function signature.